pkgs/by-name/un/unofficial-homestuck-collection/package.nix

```
@@ -0,0 +1,34 @@
+{
+  appimageTools,
+  lib,
+  fetchurl,
+}:
+let
+  pname = "unofficial-homestuck-collection";
+  version = "2.5.7";
+  src = fetchurl {
+    url = "https://github.com/homestuck/unofficial-homestuck-collection/releases/download/${version}/The-Unofficial-Homestuck-Collection-${version}.AppImage";
+    hash = "sha256-Nd+Uf3HY8MNx/8IZW5lLqsead6LMptjyVT1tTfl8K1A=";
+  };
+  contents = appimageTools.extract { inherit pname version src; };
+in
+appimageTools.wrapType2 rec {
```

this rec is unnecessary 
```suggestion
appimageTools.wrapType2 {
```